lib/test/test_hamburger.rb

```diff
@@ -0,0 +1,272 @@
+require_relative '../../lib/cdo/hamburger'
+require 'minitest/autorun'
+
+class I18n
+  def self.t(key)
+    key
+  end
+end
+
+class CDO
+  def self.code_org_url(path)
+    path
+  end
+
+  def self.studio_url(path)
+    path
+  end
+end
+
+class Level
+  def report_bug_url(request)
+    "url"
+  end
+
+  def try(property)
+    false
+  end
+end
+
+class LevelAppLab
+  def report_bug_url(request)
+    "url"
+  end
+
+  def try(property)
+    property == :is_project_level
+  end
+
+  def game
+    "AppLab"
+  end
+end
+
+class LevelGameLab
+  def report_bug_url(request)
+    "url"
+  end
+
+  def try(property)
+    property == :is_project_level
+  end
+
+  def game
+    "GameLab"
+  end
+end
+
+class Game
+  def self.gamelab
+    "GameLab"
+  end
+
+  def self.applab
+    "AppLab"
+  end
+end
+
+class HamburgerTest < Minitest::Test
+  # Visibility CSS class tests.
+
+  def test_level_teacher_en
+    visibility = Hamburger.get_visibility({level: true, user_type: "teacher", language: "en"})
+
+    assert visibility[:hamburger_class]         == Hamburger::SHOW_ALWAYS
```

Comment:
Prefer [`assert_equal`](http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-assert_equal) - gives better assertion errors when it fails.  You can still whitespace format as you do here.  See [Minitest Assertions documentation](http://docs.seattlerb.org/minitest/Minitest/Assertions.html) for a complete list of possibilities.